src/Runtime/Runtime/System.Windows/TextMeasurementService.cs

```diff
@@ -72,11 +72,28 @@ internal class TextMeasurementService : ITextMeasurementService
         private string measureTextBoxElementID;
         private string measureTextBlockElementID;
 
+        private double savedTextBlockFontSize;
+        private string savedTextBlockFontFamily;
+        private FontStyle savedTextBlockFontStyle;
+        private FontWeight savedTextBlockFontWeight;
+        private TextWrapping savedTextBlockTextWrapping;
+        private Thickness savedTextBlockPadding;
 
         public TextMeasurementService()
         {
             measureTextBoxElementID = "";
             measureTextBlockElementID = "";
+
+            savedTextBlockFontSize = 0;
+            savedTextBlockFontFamily = "";
+#if OPENSILVER
+            savedTextBlockFontStyle = new FontStyle(-1);
+#else
+            savedTextBlockFontStyle = new FontStyle();
+#endif
```

Comment:
* -1 is not a supported value for FontStyle's constructor
* Use FontStyle enum in UWP configuration
```suggestion
#if MIGRATION
            savedTextBlockFontStyle = FontStyles.Normal;
#else
            savedTextBlockFontStyle = FontStyle.Normal;
#endif
```